plenum/common/messages/node_messages.py

```diff
@@ -215,6 +219,19 @@ class Reply(MessageBase):
     )
 
 
+class ActionResult(MessageBase):
```

Comment:
Such a subclass of MessageBase is not needed. Actually `FieldValidator` and its subclasses (`MessageBase` is one of them) are used for validation against schemas. Now `ActionResult` is not used in any schemas since `Reply.result` is defined as `AnyValueField`.

Moreover, `Reply.result` for an action should be just a dictionary with the specific fields, it should not be a simple class, because it must be transferred via network as a dictionary. For example, see `Node.handle_get_txn_req` - just a dictionary is used as `Reply.result`. (If you saw at `MessageBase` subclasses, please note that `MessageBase` is a subclass of `Mapping`, so `MessageBase` provides conversion to a dictionary. But in our case we should use just a dictionary because we need not validation facilities for the reply content.)

Reply:
If ActionResult should be a dictionary and shouldn't contains fields. Do we need this class? Can we remove it? At the time of the Reply creation, we will just put the dictionary with keys and values from the next comment.

Reply:
Yes, we should remove this class and put the dictionary as `result` to `Reply`.